src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  margin-top: 16px;
+  align-items: flex-end;
+  text-align: justify;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
+  }
+
+  &__star:nth-child(-n + 1),
+  &__star:nth-child(-n + 2),
+  &__star:nth-child(-n + 3),
+  &__star:nth-child(-n + 4) {
```

Comment:
The selectors '&__star:nth-child(-n + 1)', '&__star:nth-child(-n + 2)', '&__star:nth-child(-n + 3)', and '&__star:nth-child(-n + 4)' are redundant. You can simplify this by using '&__star:nth-child(-n + 4)' alone to target the first four children.